JWPlayer-SDK-iOS-Demo/SwiftViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SwiftViewController.swift
+//  JWPlayer-SDK-iOS-Demo
+//
+//  Created by Amitai Blickstein on 2/26/19.
+//  Copyright © 2019 JWPlayer. All rights reserved.
+//
+
+import UIKit
+
+class SwiftViewController: UIViewController {
+    @IBOutlet weak var playerContainerView: UIView!
+    var player: JWPlayerController?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        player = JWPlayerFactory.newPlayer()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        layout(playerView: player!.view)
+    }
+
+    func layout(playerView: UIView) {
+        playerContainerView.addSubview(playerView)
+        playerView.constrainToSuperview()
+    }
+}
+
+
+// MARK: - Helper method
+
+extension UIView {
+    /// Constrains the view to its superview, if it exists, using Autolayout.
+    /// - precondition: For player instances, JWP SDK 3.3.0 or higher.
+    @objc func constrainToSuperview() {
+        translatesAutoresizingMaskIntoConstraints = false
+        let horizontalConstraints = NSLayoutConstraint.constraints(withVisualFormat: "H:|[thisView]|",
+                                                                   options: [],
+                                                                   metrics: nil,
+                                                                   views: ["thisView": self])
+        
```

Comment:
@karimMourra This is the pattern used in {{JWFreewheelVideoBaseView.m}}, line ~162.